build-scripts/ubuntu-1604/build-3rd-parties.sh

```diff
@@ -83,3 +83,9 @@ build_from_pypi intervaltree 2.1.0
 build_from_pypi jsonpickle 0.9.6
 # TODO: add libsnappy dependency for python-rocksdb package
 build_from_pypi python-rocksdb 0.6.9
+
+build_from_pypi wcwidth 
+build_from_pypi ujson 1.33
```

Comment:
we already has the same version of this package in setup.py and 'xenial' version is used